docs/Indexing_JSON.md

```diff
@@ -134,7 +131,7 @@ The LOAD parameters accepts JSON Path expressions. Any value (even not indexed)
 In this example we are loading two numeric values from the JSON document applying a simple operation.
 
 ```
-FT.AGGREGATE myIdx '*' LOAD 6 $.user.hp AS hp $.user.dmg AS dmg APPLY '@hp-@dmg' AS points
+FT.AGGREGATE userIdx '*' LOAD 6 $.user.hp AS hp $.user.dmg AS dmg APPLY '@hp-@dmg' AS points
```

Comment:
Written above:

> Any value (even not indexed) can be used in the pipeline.

Is it a "value" (which is suitable for Hash) or actually a "property" (relevant for JSON)?

Reply:
For later, need to decide what would be the best term for "value"